examples/web/client/src/components/MakeRequests.js

```diff
@@ -3,18 +3,15 @@ import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import { succeedAlways, succeedSometimes, failSometimes } from '../actions';
 
+const noop = () => {};
+
 class MakeRequests extends React.Component {
 
   buildHandler = (handler) => {
     const { successCallback, errorCallback } = this.props;
-    let promise;
-    if (successCallback) {
-      promise = handler().then(successCallback);
-    }
-    if (errorCallback) {
-      promise.catch(errorCallback);
-    }
-    return promise || handler();
+    return handler()
+      .then(successCallback || noop)
+      .catch(errorCallback || noop);
```

Comment:
`noop()` is not needed, if the arguments for `then()` or `catch()` are `null` or `undefined`, then they are not executed. It's part of the spec.

Reply:
so instead it would be `.then().catch()`?

Reply:
Oh! I did not know that. Nifty.

Reply:
Good you like it :-)